vehicle/mkvi/software/throttle/throttle.yml

```diff
@@ -0,0 +1,83 @@
+name: throttle
+subscribe:
+  # Transmit MOB is 0
+  - name: brakelight
+    mob: 1
+  - name: dashboard
+    mob: 2
+publish:
+  - name: M192_Command_Message
+  - name: throttle
+    id: 0xC
+    freq_hz: 100
+    signals:
+      - name: throttle_state
+        slice: 0 + 8
+        unit:
+          type: enum
+          values:
+            - THROTTLE_IDLE
+            - THROTTLE_RUN
+            - THROTTLE_OUT_OF_RANGE
+            - THROTTLE_POSITION_IMPLAUSIBILITY
+            - THROTTLE_BRAKE_PRESSED
+      - name: throttle_1_pos
+        slice: 8 + 16
+        unit:
+          type: int16_t
+          name: '%' 
+          offset: 0
+          scale: 100/255
+      - name: throttle_2_pos
+        slice: 24 + 16
+        unit:
+          type: int16_t
+          name: '%'
+          offset: 0
+          scale: 100/255
+      - name: ss_is
+        slice: 41 + 1
+        unit:
+          type: bool
+          values:
+            t: CLOSED
+            f: OPEN
+  - name: throttle_debug
+    id: 0x5C0
```

Comment:
I think this ID should be 0x6C0 (based on https://docs.google.com/spreadsheets/d/12KenzPC5moFQULhOhoBm-pcmC4ligmMhP_M-TPqN-CI/edit#gid=1599758060)